### ISSUE TITLE
`Cannot read property 'kind' of null` when generate swift API

### ISSUE THREAD

**Issue by beeth0ven** [apollographql/apollo-codegen#407]
Thanks for creating this useful tool, it saves my life.

I've a small question with follow use case:

### typeDefs.js:

```
  type Query { 
    rankedMedia(cursor: Float): CursorMedia!
    ...
 }

...
```

### ViewController.graphql:
```
query RankedMedia {
    rankedMedia(cursor: null) {
        cursor
        items {
            _id
        }
    }
}
```

When i generate the `API.swift` it throws a error `Cannot read property 'kind' of null`.

Since i pass `null` as variable in `ViewController.graphql` which is not handled by apollo-codegen:
https://github.com/apollographql/apollo-codegen/blob/master/src/swift/helpers.ts#L199

Is it by design or shall we add the handle null logic here?

**Comment by martijnwalraven** [comment]
No, good catch, we should definitely check whether `value` is `null` there to handle literal `null` values.

As an aside, there doesn't seem to be any need in your case to pass an explicit `cursor: null` to `rankedMedia`.

**Comment by beeth0ven** [comment]
Interesting, didn't know this tip.
Thanks!
